### PR TITLE
refactor: moved warning and errors for import into accordions

### PR DESCRIPTION
**Pull request by Ryoken**
#### What's this PR do?
Organizes errors and warnings after uploading a file.

#### How should this be manually tested?
See original ticket for testing instructions.

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2929

#### Screenshots (if appropriate)
Initial screen includes folded list:
Screen Shot 2021-11-02 at 4 26 44 PM

multiple file support and opened list:
Screen Shot 2021-11-02 at 4 29 46 PM

if errors exist they appear like this:
Screen Shot 2021-11-02 at 4 21 10 PM

### CODE REVIEW

**Review by macintoshpie — COMMENTED** [review]
The accordions look great, much cleaner.

I tested uploading a "bad" buildingsync file by editing ex_1.xml (I wrote "Bogus" in the `auc:ClimateZone` element) and it looks like we might need to handle this case (see here https://github.com/SEED-platform/seed/blob/15bddd41aaa855819a3d875dd126d2b69cc59c50/seed/static/seed/partials/data_upload_modal.html#L355)

I think step 12 is generally when a file completely fails upload? (which I guess we trigger if any errors occur in validation?)

Could you look into that?

**Comment by Ryoken** [comment]
I split it out into a partial so now the other failing section now renders the same way.

Screen Shot 2021-11-03 at 12 12 28 PM

File: seed/static/seed/partials/data_upload_modal.html [hunk 1]
```diff
@@ -318,29 +315,14 @@ <h5>Use Case Warnings</h5>
...
-                            </div>
-                        </div>
-                    </span>
-                </div>
+                <p ng-if="!step_12_buildingsync_validation_error" translate>An error occurred while processing the file. Please ensure that your file meets the required specifications.</p>
+                <pre class="text-wrap" ng-class="{'pre-scrollable': step_12_error_message.length >= 300}" ng-show="step_12_error_message && !step_12_buildingsync_validation_error">{$ step_12_error_message | json $}</pre>
+                <span ng-if="step_12_buildingsync_validation_error">
+                    <div class="alert alert-danger">BuildingSync file(s) failed to validate. No files were imported. Please fix the errors and re-upload the corrected file(s).</div>
+                    <div ng-repeat="file_issues in buildingsync_issues">
+                        <ng-include src="::urls.static_url + 'seed/partials/import_messages.html'"></ng-include>
```
Comment (macintoshpie) [inline seed/static/seed/partials/data_upload_modal.html:323]: noice, v clean 💯

**Review by macintoshpie — APPROVED** [review]
Great! I tested use case warnings/errors and schema errors
Screen Shot 2021-11-03 at 12 29 35 PM
Screen Shot 2021-11-03 at 12 28 58 PM

**Pull request merged**